CMakeLists.txt

```diff
@@ -184,6 +184,12 @@ endif()
 
 find_package(Threads REQUIRED)
 
+if (ANDROID)
+  find_package(openssl REQUIRED CONFIG)
+else ()
+  find_package(OpenSSL REQUIRED)
+endif ()
```

Comment:
OpenSSL is not generally required, it's required for the server and optional for the client.
Please remove.

Reply:
@xabolcs  there is a branch that makes SSL optional for the server: https://github.com/badaix/snapcast/tree/optional_ssl (`-DBUILD_WITH_SSL=<ON|OFF>`, default `ON`).

I think I will merge it tomorrow into develop and with the next release to master.